### ISSUE TITLE
Still crashes to desktop on "loading world" even with latest git.froggi.es version

### ISSUE THREAD

**Issue by Satv9** [Joshua-Ashton/VPhysics-Jolt#82]
Replaced physics dll file, disabled then UNSUBSCRIBED from all addons, even got latest development version and replaced physics dll with that, but it still crashes instantly when trying to start singleplayer session.

Will I just have to wait for 0.11? Considering latest didn't work I can't even be sure that will either. I'm on windows x64.

**Comment by DoomGuyCooming** [comment]
Same problem,I guess we need to wait for version 0.11

**Comment by ScottVordem** [comment]
Are you in the x86_64 branch in Garry's mod? This only works with 64-bit Garry's Mod.

**Comment by Satv9** [comment]
Yes

**Comment by Bollinha** [comment]
I put the dll files, renamed the files to vphysics, started gmod in 86x64 version, entered the game, and the game did not open, it gave an error:
"App Framework: U-nable to load module vphysics.dll!"

**Comment by DoomGuyCooming** [comment]
Version 0.11 is released,use it

**Comment by Joshua-Ashton** [comment]
Can you try installing the Visual Studio 2022 C++ runtime?

**Comment by Bollinha** [comment]
> Can you try installing the Visual Studio 2022 C++ runtime?

no, I hadn't installed it, I installed it and it works perfectly now. thanks and sorry for having to bother you.
